src/test/kotlin/lotto/WinningStaticsTest.kt

```diff
@@ -5,25 +5,91 @@ import io.kotest.matchers.shouldBe
 import lotto.domin.LottoNumberSet
 import lotto.domin.LottoWinningAmount
 import lotto.domin.WinningStatics
+import lotto.dto.InputLottoNumberDto
 import lotto.dto.InputPaymentRequestDto
-import lotto.dto.InputWinningLottoNumberDto
 
 class WinningStaticsTest : FreeSpec({
 
     "result" - {
 
-        "당첨 로또 개수와 수익률을 반환한다." {
+        "당첨된 로또가 없는 경우 결과를 반환한다." {
+            val lottoNumberSet = LottoNumberSet(listOf(13, 14, 15, 16, 17, 18))
+            val lottoRecord = listOf(lottoNumberSet)
+            val paymentDto = InputPaymentRequestDto.convertPayment("14000")
+            val winningLotto = InputLottoNumberDto.of(listOf("1", "2", "3", "10", "11", "12"), "13")
+            val winningStat = WinningStatics(paymentDto, winningLotto, lottoRecord)
+
+            val winningResult = winningStat.result()
+
+            LottoWinningAmount.values().forEach { it ->
+                winningResult.winningLottoList[it] shouldBe 0
+            }
+            winningResult.profitRatio shouldBe 0
+        }
+
+        "당첨 로또가 3개인 경우의 결과를 반환한다." {
```

Comment:
```suggestion
        "당첨 로또가 3개인 경우 5등입니다." {
```
등수를 명시해주면 좀 더 이해하기 편할 것 같아요. (이하 유사 관련 내용 동일)